src/views/Home/components/DiningBalance/index.jsx

```diff
@@ -11,7 +11,7 @@ import {
 import EditIcon from '@mui/icons-material/Edit';
 import GordonLoader from 'components/Loader';
 import { useEffect, useState } from 'react';
-import { Doughnut } from 'react-chartjs-2';
+import { Pie } from 'react-chartjs-2';
```

Comment:
Why is this unrelated change in this pull request.  Please put it in a different branch and make a pull request just for it.  Then it won't need to wait for the other parts you are still working on.